cou/exceptions.py

```diff
@@ -33,27 +33,31 @@ def __init__(self, cmd: str, code: str, output: str, err: str):
 
 
 class UnitNotFound(Exception):
-    """Unit not found in actual dict."""
+    """Exception raised when unit not found in the actual dict."""
```

Comment:
Actually now that you've modified this docstring I realize it's kinda confusing... which "actual dict"? You're also missing an article:
```suggestion
    """Exception raised when a unit is not found in the actual dict."""
```

Reply:
Reworded